### DISCUSSION TITLE
muting users I follow in my follow feed

### DISCUSSION THREAD

**Discussion by geduldig** [community/community#16860] [Other Feature Feedback, Questions, & Ideas]
I would like to suppress some users that I follow from my "follow" feed, only because they star others so frequently that it overwhelms my feed. 

Is this possible, or is this a feature request?

**Comment by siminapasat** [comment]
We're working on this as we speak @geduldig . Thank you for your suggestion!

**Comment by jfullstackdev** [comment]
there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
